File: java-apirest/javapy/src/teste-desempenho/SumTest.java
public class SumTest {
    public static long calculateSum(int n) {
        long sum = 0;
        for (int i = 1; i <= n; i++) {
            sum += i;
        }
        return sum;
    }

    public static void main(String[] args) {
        int n = 1000000000; // Defina o tamanho do teste

        long startTime = System.currentTimeMillis();
        long totalSum = calculateSum(n);
        long endTime = System.currentTimeMillis();

        System.out.println("Soma: " + totalSum);
        System.out.println("Tempo de execução do Java: " + (endTime - startTime) / 1000.0 + " segundos");
    }
}

File: java-apirest/javapy/src/teste-desempenho/sum_test.py
# .py
import time

def calculate_sum(n):
    return sum(range(1, n + 1))

# Defina o tamanho do teste
n = 1000000000

start_time = time.time()
total_sum = calculate_sum(n)
end_time = time.time()

print(f"Soma: {total_sum}")
print(f"Tempo de execução do Python: {end_time - start_time} segundos")